config/locales/javascript/javascript.en.yml

```diff
@@ -239,9 +239,9 @@ en:
       contacts: "Contacts"
       posts: "Posts"
 
-    conversation:
-      new:
-        no_contacts: "You need to add some contacts before you can start a conversation."
```

Comment:
You are still removing the translation here and adding it to `config/locales/diaspora/en.yml ` instead.